src/network/MissionControlProtocol.h

```diff
@@ -29,15 +30,22 @@ class MissionControlProtocol : public WebSocketProtocol { // TODO: add documenta
 
 private:
 	void videoStreamTask();
+	void jointPowerRepeatTask();
 	SingleClientWSServer& _server;
 	std::shared_mutex _stream_mutex;
+	std::shared_mutex _joint_power_mutex;
```

Comment:
If there's only one reader and possibly many writers, is there any point in making this a shared mutex? There aren't any concurrent readers to benefit from anyway, so imo a regular mutex would do the same job just fine, with less overhead.

Reply:
Yeah, you're probably right, I think I was just copying the stuff I already wrote for the stream mutex without much thought lol